### PR TITLE
add intermediate result EP

### PR DESCRIPTION
**Pull request by anmol-nura**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2239,6 +2240,653 @@ The `Errors` field in the JSON will have the following format:
...
+            Authorization: Bearer <api token>
+
++ Response 200 (application/json)
+
+       {
+    "MeasurementID": "663b8ac1-67b5-4a2a-939b-6bdae2abca32",
+    "MeasurementDataID": "663b8ac1-67b5-4a2a-939b-6bdae2abca32:1",
+    "MeasurementResultID": "9f8ae87c-b899-45ce-8b5f-8e57f753cfa9",
+    "Multiplier": 10000,
+    "Channels": {
```
Comment (daphnezhou) [inline apiary.apib:2266]: Please only include `HR_BPM` and `SNR` in the results as an example.

**Review by daphnezhou — CHANGES_REQUESTED** [review]

File: apiary.apib (hunk 1 above)
Comment (anmol-nura) [reply apiary.apib:2266]: I don't have that Examples. Do you have one?

Comment (daphnezhou) [reply apiary.apib:2266]: It's actually in your current example, but I pasted it here again:

"SNR": {
            "Data": [
                1000000
            ],
            "Channel": "slow/toi4"
        }
"HR_BPM": {
            "Data": [
                909906
            ],
            "Channel": "slow/toi4"
        }

**Review by daphnezhou — CHANGES_REQUESTED** [review]

**Review by daphnezhou — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -2239,6 +2240,146 @@ The `Errors` field in the JSON will have the following format:
 
 + Response 200 (application/json)
 
+
+### Intermediate Results [GET /measurements/{ID}/results/{chunkOrder}]
+
+This endpoint is used to get the intermediate results of a measurement done. The intermediate results are valid for 2 minutes after the measurement is done.
```
Comment (Vineet-nuralogix) [inline apiary.apib:2246]: This endpoint is used to get intermediate results of a measurement.

**Pull request merged**